etl-jobs/src/main/scala/org/sunbird/learner/jobs/SelfDeclarationToConsentMigration.scala

```diff
@@ -0,0 +1,59 @@
+package org.sunbird.learner.jobs
+
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+import org.apache.spark.storage.StorageLevel
+
+object SelfDeclarationToConsentMigration extends Serializable {
+    private val config: Config = ConfigFactory.load
+    
+    def main(args: Array[String]): Unit = {
+        implicit val spark: SparkSession =
+            SparkSession
+                .builder()
+                .appName("SelfDeclarationToConsentMigration")
+                .config("spark.master", "local[*]")
+                .config("spark.cassandra.connection.host", config.getString("spark.cassandra.connection.host"))
+                .config("spark.cassandra.output.batch.size.rows", config.getString("spark.cassandra.output.batch.size.rows"))
+                .config("spark.cassandra.read.timeoutMS", config.getString("spark.cassandra.output.batch.size.rows"))
+                .getOrCreate()
+        val res = time(migrateSelfDeclaredToConsent());
+        Console.println("Time taken to execute script", res._1);
+        spark.stop();
+    }
+    
+    def migrateSelfDeclaredToConsent()(implicit spark: SparkSession): Unit = {
+        var userConsentdDF = spark.read.format("org.apache.spark.sql.cassandra").option("keyspace", "sunbird").option("table", "user_consent").load().persist(StorageLevel.MEMORY_ONLY)
+        userConsentdDF = userConsentdDF.where(col("object_type").isin("global", "Organisation"))
+        println("user_consent data Count : " + userConsentdDF.count())
+        var userDeclaredDF = spark.read.format("org.apache.spark.sql.cassandra").option("keyspace", "sunbird").option("table", "user_declarations").load().persist(StorageLevel.MEMORY_ONLY)
+        println("user_declarations data Count : " + userDeclaredDF.count())
+        userDeclaredDF = userDeclaredDF.join(userConsentdDF, userDeclaredDF.col("userid") === userConsentdDF.col("user_id"), "leftanti")
+        println("userDeclaredDF after join latest data Count : " + userDeclaredDF.count())
```

Comment:
Use persist everywhere if you are getting counts

Reply:
@SanthoshVasabhaktula : done.